Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthRequestParams.kt

```diff
@@ -0,0 +1,10 @@
+package com.braintreepayments.api.venmo
+
+import com.braintreepayments.api.BrowserSwitchOptions
+
+/**
+ * Used to request Venmo authentication via [VenmoLauncher.launch] )}
+ */
+class VenmoPaymentAuthRequestParams internal constructor(
```

Comment:
Seems like we don't have consistency across modules on these `PaymentAuthRequestParams` classes now. I created a separate ticket to align them, but maybe we can start the discussion on the approach here. Should these all be data classes? (cc @tdchow) I also think `browserSwitchOptions` can be library scope on all of them

Reply:
Great catch! I think we'll want to match the other modules in making this a sealed class.